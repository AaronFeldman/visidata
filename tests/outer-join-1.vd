sheet	col	row	longname	input	keystrokes	comment
			open-file	tests/data1.tsv	o	
			open-file	tests/data2.tsv	o	
data1	Key		key-col		!	
data2	Key		key-col		!	
data2			sheets-stack		S	
sheets		キdata2	select-row		s	
sheets		キdata1	select-row		s	
sheets		キdata2	slide-down		J	
sheets			join-sheets	outer	&	
data1+data2	A		key-col		!	
data1+data2			sort-keys-asc		g[	
data1+data2	A		key-col		!	
