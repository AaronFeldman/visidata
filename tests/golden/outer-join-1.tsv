Key	data1_A	data1_B	data2_C	data2_D
1	a1	b1		
2	c1	d1	a2	b2
2	e1	f1	a2	b2
